src/modules/main/store/actionTypes.ts

```diff
@@ -6,11 +6,23 @@ export enum LoadData {
   LOAD_EXPERTS = 'LOAD_EXPERTS',
 }
 
-export interface INewestItem {}
+export interface IExpertsItem {}
+
+export interface IMeta {
```

Comment:
Temporary metadata, just to show component functionality.

Reply:
bear in mind that this will probably differ from the actual BE endpoint implementation.

you can check the output format with BE team.
most probably, if implementing the pagination on the BE, `meta` looks like following
```
{
length: number, // amount of returned items
total: number, // amount of total items
page: number, // current page index
totalPages?: number, // total pages
}
```
- these are most common fields.

therefore,
```
limit?: number;
showMore: boolean;
```
- will be calculated on FE side